### PR TITLE
README Suggestions

### PR DESCRIPTION
**Pull request by mole99**
Some suggestions to the README :)

Just comment if something is unclear.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,8 +21,8 @@ If your team wants to use this, follow [Getting started](#getting-started).
 - Test summary with errors from all test runs
 - Configure how debug logs get parsed and interpreted
 - Fancy progress bar
-- Uses qemu snapshots
-- Individual test configuration via a test headers (timeout, expected exit codes)
+- Uses qemu snapshots for faster bootup
```
Comment (PaideiaDilemma) [inline README.md:24]: I would suggest `- Uses qemu snapshots to avoid booting multiple times` or removing this point.

File: README.md [hunk 2]
```diff
@@ -47,7 +47,9 @@ If your team wants to use this, follow [Getting started](#getting-started).
 
 Check out the the latest run of the
 [demo workflow](https://github.com/PaideiaDilemma/Tortillas/actions/workflows/demo_ci.yml?query=branch%3Amain),
-for examples of tortillas in action.
+for examples of tortillas in GitHub Actions.
+
+If you create a working GitLab CI workflow, we'd be happy if you could contribute it to Tortillas.
```
Comment (PaideiaDilemma) [inline README.md:52]: Here I actually was trying to say "tortillas in action".
The new note below does not belong in the demo section I think.
Also I think I am still going to add our gitlab ci file in an adapted form as an example.

File: README.md [hunk 3]
```diff
@@ -125,14 +127,10 @@ Verify the setup, by [running tortillas](#running-tortillas). \
 You can now check out [Test specifications](#test-specifications) and add test specifications
 to your tests.
 
-### CI/CD
-[TODO]
-
```
Comment (PaideiaDilemma) [inline README.md:130]: This is were I thought we could add a rough outline of how to set up gitlab and github CI/CD.

File: README.md (hunk 1 above)
Comment (mole99) [reply README.md:24]: Good point!

File: README.md (hunk 2 above)
Comment (mole99) [reply README.md:52]: I see, I adjusted it a bit. Feel free to change it again ;)

File: README.md (hunk 3 above)
Comment (mole99) [reply README.md:130]: Ups, revert!

**Comment by mole99** [comment]
I think it can be merged now :+1:

**Pull request merged**